src/index.ts

```diff
@@ -38,49 +43,89 @@ const argv = yargs(process.argv.splice(2))
 	},
 	o: {
 		alias: 'open',
+		group: 'General Options:',
 		desc: 'Open in browser',
 		type: 'boolean',
 		default: false
 	},
 	i: {
 		alias: 'init',
+		group: 'General Options:',
 		desc: 'Init serve project',
 		type: 'boolean',
 		default: false
 	},
-	x: {
-		alias: 'extensions',
-		desc: 'File extension fallbacks: If a file is not found, search for files with the specified extensions',
-		type: 'array',
-		default: ['html', 'htm'],
-		coerce(arg: any[]) {
-			arg.forEach(v => {
-				const type = typeof v
-				if (type !== 'string') throw new Error(`File extensions must be strings. [${v}] is of type '${type}'`)
-			})
-			return arg as string[]
-		}
-	},
 	e: {
 		alias: 'err-page',
+		group: 'General Options:',
 		desc: 'Path to the file that server will respond with if an Internal Server Error occurs. Priority: arg > ./.500.html > SCRIPT_DIR/assets/500.html',
 		type: 'string',
 		default: null,
 		defaultDescription: '.500.html'
 	},
 	n: {
 		alias: 'not-found-page',
+		group: 'General Options:',
 		desc: 'Path to the file that server will respond with if requested path is not found. Priority: arg > ./.404.html > SCRIPT_DIR/assets/404.html',
 		type: 'string',
 		default: null,
 		defaultDescription: '.404.html'
 	},
 	r: {
 		alias: 'routes',
+		group: 'General Options:',
 		desc: 'Path to routes map. Priority: arg > ./.routes',
 		type: 'string',
 		default: null,
 		defaultDescription: '.routes'
+	},
+	'sd': {
+		alias: 's-dotfiles',
+		group: 'Static Options:',
+		desc: 'Determines how dotfiles (files or directories that begin with a dot ".") are treated',
+		type: 'string',
+		choices: ['allow', 'ignore', 'deny'],
+		default: 'ignore',
+		coerce(arg: string) {
+			if (!['allow', 'ignore', 'deny'].includes(arg))
+				throw new Error(`Param "--s-dotfiles" must be one of the following: "allow" | "ignore" | "deny"`)
```

Comment:
No need to use template literal here.
```suggestion
				throw new Error('Param "--s-dotfiles" must be one of the following: "allow" | "ignore" | "deny"')
```